src/agent.rs

```diff
@@ -5,7 +5,7 @@ asset, and other news (random noise in our case).
 TODO: Fundamentalists? From the DGA paper.
 */
 
-use rand::random;
+use rand::prelude::*;
```

Comment:
Always prefer direct imports over globs.

Reply:
In vscode you can press ctr+. when over the glob import and it will show a hint where you can resolve it automatically.
Better to do manually, since it will then import from the prelude.